src/Serilog.Enrichers.Environment/EnvironmentLoggerConfigurationExtensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright 2013-2016 Serilog Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Serilog.Configuration;
+using Serilog.Enrichers;
+
+namespace Serilog
+{
+    public static class EnvironmentLoggerConfigurationExtensions
+    { 
+        /// <summary>
+        /// Enrich log events with a MachineName property containing the current <see cref="Environment.MachineName"/>.
```

Comment:
Missing an XML comment here. Created #675 to cover this as there are already a few merged PRs that missed this. Easier to go back over it all in one hit.